### DISCUSSION TITLE
Support for seeding database

### DISCUSSION THREAD

**Discussion by jonny7** [community/community#26532] [Actions]
<p>I had put a question into the actions team regarding seeding a database and was told that currently there isn’t a mechanism for this natively. So they suggested I either put an issue in the examples actions site or ask in the forum to see for available work arounds.</p>
<p>I was basically attempting the equivalent of dockers</p>
<pre><code>psql:
    image: "postgres"
    volumes:
      - ./seed.sql:/docker-entrypoint-initdb.d/init.sql
</code></pre>
<p>I wondered what is the best way currently, and is there any kind of plan to support native seeding.</p>
<p>Thanks :slight_smile:</p>

**Comment by Yanjingzhu** [comment, marked as answer]
<p>Hi <a class="mention" href="/jonny7">@jonny7</a> ,</p>
<p>To be honest , I am not good at database seeding. I will contact our engineering team for help. </p>
<p>And you could consider creating an issue here <a href="https://github.com/actions/example-services/issues" rel="nofollow noopener">https://github.com/actions/example-services/issues</a> asking an example for postgres satabase seeding. </p>
